src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,152 @@
+    @Inject
+    private BetDao betDao;
+    @Setter
+    private AuthorizationService authService;
+    @Setter
+    private RegistrationService regService;
+    @Setter
+    private Mapper<User> userMapper;
+    @Setter
+    private Mapper<Bet> betMapper;
+
+    public void handle() {
+        String cursor = CURSOR_TEXT;
+        User user = null;
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.print(cursor);
+            String line = scanner.nextLine();
+            String[] data = line.split(" ");
+            try {
+                switch (data[0]) {
+                    case "/q":
+                        return;
+                    case "/l":
+                        user = authorize(data);
+                        cursor = "\n[" + user.getLogin() + "] enter bet->";
+                        continue;
+                    case "/r":
+                        register(data);
+                        continue;
+                    case "/h":
+                        outputCommands();
+                        continue;
+                    case "/mb":
+                        outputUserBets(user);
+                        continue;
+                    case "/signout":
+                        user = signOut(user);
+                        cursor = CURSOR_TEXT;
+                        continue;
+                    default:
+                        if (user != null) {
+                            placeBet(data, user);
+                        }
+                }
+            } catch (WrongLoginOrPasswordException e) {
+                System.out.println("Wrong login or password");
+            } catch (InvalidAgeException e) {
+                System.out.println("For adults only (18+)");
+            } catch (DuplicateUserException e) {
+                System.out.println("This login is busy");
+            } catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+                System.out.println("Invalid data");
+            } catch (InvalidLoginStringException e) {
+                System.out.println("Invalid string. Example: [/l login password]");
+            } catch (InvalidStringRegisterException e) {
+                System.out.println("Invalid string. Example: [/r login password age]");
+            } catch (UserNotAuthorizedException e) {
+                System.out.println("You are not logged in");
+            } catch (InvalidStringBetException e) {
+                System.out.println("Invalid string. Enter value and risk, "
+                        + "example: 100 0.1");
+            }
+        }
+    }
+
+    private void placeBet(String[] data, User user) throws InvalidStringBetException {
+        if (data.length != 2) {
+            throw new InvalidStringBetException("Invalid string. Enter value and risk, "
+                    + "example: 100 0.1");
+        }
```

Comment:
now there are too many exceptions :)

don't replace the conditional logic with exceptions in your code

I would suggest to remove the redundant Exceptions and in case of any wrong condition just sout the necessary message and `return` from a private method